Automattic/wp-calypso
# Add oneClickUpsell A/B test

#### Changes proposed in this Pull Request

* Adds oneClickUpsell A/B test. See pcbrnV-aX-p2 for more details. 

#### Testing instructions



* Make sure you're assigned to `test` group of `oneClickUpsell` abtest.
* Purchase Personal or Premium dotcom plan for your website.
* You may need to remember the receipt ID.
* Visit `/checkout/offer-quickstart-session/RECEIPT_ID/YOUR_SITE_SLUG`. Please replace RECEIPT_ID and YOUR_SITE_SLUG with your owns. It would look like `/checkout/offer-quickstart-session/1234567/testblog.woordpress.com`.
* Ensure you have one or more stored credit cards.
* Click the "Yes, I want a WordPress Expert by my side!" button.
* You should be able to see a modal dialog as below:
  <img width="788" alt="Checkout_‹_Quick_Start_Session_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/94092862-c9880a80-fe56-11ea-9e7d-1260138bee12.png">
* Click on the Pay button and the transaction of purchasing Concierge Session is going to be _automatically_ processed.
* When everything goes okay, a sucess notification appears on the homepage of your site. Otherwise, an error notification appears.
  <img width="961" alt="My_Home_‹_Site_Title_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/93323074-17958080-f84f-11ea-9d65-54925c3ad644.png">
